src/packages/@ncigdc/containers/explore/presetFacets.ts

```diff
@@ -116,14 +116,157 @@ export const clinicalFacets: IClinicalFacetProps[] = [
   },
 ];
 
-export const customSorting = {
-  'diagnoses.ajcc_clinical_t': -23,
-  'diagnoses.ajcc_clinical_n': -22,
-  'diagnoses.ajcc_clinical_m': -21,
-  'diagnoses.ajcc_clinical_stage': -20,
-  'diagnoses.ajcc_pathologic_t': -13,
-  'diagnoses.ajcc_pathologic_n': -12,
-  'diagnoses.ajcc_pathologic_m': -11,
-  'diagnoses.ajcc_pathologic_stage': -10,
-  'diagnoses.ajcc_staging_system_edition': -1,
+const clinicalFacetSortingDictionary = {
```

Comment:
This dictionary will allow for customised sorting without having to change the numbers manually, unlike i had designed it initially. Just add a new value exactly where you want it, and Bob's your uncle 👍